README.md

```diff
@@ -69,4 +69,5 @@ Alternatively, using miniconda:
 
 #### 8. Connect to the Hub from web interface (BioThings Studio)
 
-Navigate to https://studio.biothings.io/ and create a connection to http://localhost:HUB_API_PORT
+Navigate to https://studio.biothings.io/ and create a connection to `http://localhost:HUB_API_PORT`,
```

Comment:
By default github renders this URL with actual link, which is invalid. So I disabled it and added some explanation on how it should be used.